pkg/wmi/wmi.go

```diff
@@ -0,0 +1,55 @@
+// +build windows
+
+package wmiutil
+
+import (
+	"fmt"
+
+	"github.com/StackExchange/wmi"
+)
+
+const (
+	FeatureMicrosoftHyperV    = "Microsoft-Hyper-V"
+	FeatureMicrosoftHyperVAll = "Microsoft-Hyper-V-All"
+)
+
+type FeatureInstallState uint32
+
+const (
+	FeatureInstallStateEnabled FeatureInstallState = iota + 1
+	FeatureInstallStateDisabled
+	FeatureInstallStateAbsent
+	FeatureInstallStateUnknown
+)
+
+type win32_optionalfeature struct {
+	InstallState FeatureInstallState
+}
+
+func (st FeatureInstallState) String() string {
+	switch st {
+	case 1:
+		return "enabled"
+	case 2:
+		return "disabled"
+	case 3:
+		return "absent"
+	default:
+		return "unknown"
```

Comment:
in other String() methods default returns <unknown>

Reply:
done